### ISSUE TITLE
Environment recommendation for dev, val and prod deployment

### ISSUE THREAD

**Issue by chlordk** [vitabaks/postgresql_cluster#185]
Hi

What is the recommended way for handling different environments like

1. Development
2. Valuation
3. Production

I could have a file for each of them like vars/main_dev.yml, var/main_val.yml and vars/main_prod.yml but is there a more convenient way to do it?

BTW: postgresql_cluster is an awesome ansible playbook.

**Comment by vitabaks** [comment]
Hi @chlordk 

why not just copy the entire playbook for each environment? I do this, each cluster has its own playbook fork.

**Comment by chlordk** [comment]
Good idea. In this way I would be able to try new methods without breaking prod.

Thanks.
